### PR TITLE
Updates for Dashboard to work again

### PR DESCRIPTION
**Pull request by sebradloff**
- Adds docker-compose.yml to make local dev easier
- Fixes issue with Dashboard loading
- Updates upstream docker image to inherit from

### CODE REVIEW

File: kanbantool_ticket_dash/utils.py [hunk 1]
```diff
@@ -22,7 +22,9 @@ def fetch_tickets():
     tickets = json.loads(r.text)
 
     for ticket in tickets:
-        ticket['task']['created_at'] = arrow.get(ticket['task']['created_at'])
+        # Creating task key, because the rest of the app depends on it.
```
Comment (JamesAwesome) [inline kanbantool_ticket_dash/utils.py:25]: Just add another comment line stating that this is to deal with an API change on kanbantool's side

**Review by JamesAwesome — COMMENTED** [review]
All good, just adjust the one comment so that we know what caused the extra key to be added. 👍

**Comment by sebradloff** [comment]
Cool cool. Added a comment to help clarify why the change was necessary. Lmk if there's anything else!

**Review by JamesAwesome — APPROVED** [review]

**Comment by JamesAwesome** [comment]
thanks! 🤘

**Pull request merged**